src/main/java/org/openstreetmap/atlas/checks/validation/linear/lines/GeneralizedCoastlineCheck.java

```diff
@@ -57,10 +61,8 @@ public GeneralizedCoastlineCheck(final Configuration configuration)
     @Override
     public boolean validCheckForObject(final AtlasObject object)
     {
-        return object instanceof LineItem
-                && (Validators.isOfType(object, NaturalTag.class, NaturalTag.COASTLINE)
-                        || ((LineItem) object).relations().stream().anyMatch(relation -> Validators
-                                .isOfType(relation, NaturalTag.class, NaturalTag.COASTLINE)));
+        return object instanceof LineItem && ((isCoastline(object) && isSourcePGS(object))
```

Comment:
We might want to do change ```(isCoastline(object) && isSourcePGS(object))``` to ```(isCoastline(object) && (isSourcePGS(object) || !PGSfilter))``` to avoid confusion in the case that the PGS filter is off, but isSourcePGS() returns true.